app/helpers/argo_helper.rb

```diff
@@ -9,7 +9,7 @@ module ArgoHelper
   def container_classes
     return super if controller_name == 'apo'
 
-    'container-fluid'
+    action_name == 'show' ? 'container' : 'container-fluid'
```

Comment:
This is run on every page, so I don't know why code climate sees no coverage here.